packages/cdp-metrics/src/metrics.js

```diff
@@ -0,0 +1,128 @@
+import { StorageResolution, Unit } from 'aws-embedded-metrics'
+import { MetricsHelper } from './metrics-helper.js'
+
+let logger
```

Comment:
FYI theres no need to do this with the logger as this is already being done when its created in the templates - https://github.com/DEFRA/cdp-node-frontend-template/blob/main/src/server/common/helpers/logging/logger.js#L5

so calling `server.logger` will only ever return the 1 logger